redhat_access_insights/connection.py

```diff
@@ -169,6 +153,13 @@ def get_proxies(self, config):
                 conf_proxy = scheme + location
             logger.debug("CONF Proxy: %s", conf_proxy)
             proxies = {"https": conf_proxy}
+
+        if os.environ.get('NO_PROXY') and conf_proxy:
+            logger.debug("Invalid Proxy configuration. "
+                         "You have environment variable NO_PROXY set "
+                         "as well as 'proxy' set in your configuration file. "
+                         "This is invalid. Do not expect normal results.")
```

Comment:
I think you could delete code by making config.proxy just set HTTPS_PROXY. Then NO_PROXY and config.proxy would play nice too.
`python
if config.proxy:
   os.environ.set('HTTPS_PROXY',` conf.proxy)
`